src/main/scala/org/allenai/deep_qa/experiments/datasets/OmnibusDa.scala

```diff
@@ -0,0 +1,70 @@
+package org.allenai.deep_qa.experiments.datasets
+
+import org.json4s._
+import org.json4s.JsonDSL._
+
+/**
+ * This object contains a bunch of JValue specifications for the
+ * Omnibus DA dataset's data files.
+ */
+object omnibusDaDatasets {
+
+  def omnibusDaFile(omnibusDaDir: String, grade: String, split: String): JValue = {
+    val datasetGradePath = s"Omnibus-Gr${grade}-NDDA"
+    val outputDirectory = omnibusDaDir + datasetGradePath +s"/processed/${split}/"
+    val splitCapitalized = split.capitalize
+    val inputFile = omnibusDaDir + datasetGradePath + s"/${datasetGradePath}-${splitCapitalized}.csv"
+    val outputFiles = Seq(outputDirectory + s"${split}.tsv")
+    ("sentence producer type" -> "dataset reader") ~
+      ("reader" -> "omnibus da") ~
+      ("create sentence indices" -> true) ~
+      ("input file" -> inputFile) ~
+      ("output files" -> outputFiles)
+  }
+
+  def omnibusDaDataset(omnibusDaDir: String, grade: String, split: String): JValue = {
+    val file = omnibusDaFile(omnibusDaDir, grade, split)
+    ("data files" -> List(file))
+  }
+
+  val baseDir = "/efs/data/dlfa/omnibus_ndda/"
+  // Train files
+  val omnibusNDDA04TrainFile = omnibusDaFile(baseDir, "04", "train")
```

Comment:
Also, I would probably just use `4` instead of `04`.  The `04` is for sorting on the filesystem if we ever get past grade 9.  I'm not sure it serves much purpose in a variable name.